src/scripts/main.js

```diff
@@ -1,3 +1,70 @@
 'use strict';
 
-// write code here
+const appendRow = document.querySelector('.append-row');
+const appendColumn = document.querySelector('.append-column');
+const deleteRow = document.querySelector('.remove-row ');
+const deleteColumn = document.querySelector('.remove-column');
```

Comment:
Consider using more descriptive variable names for your DOM elements. For example, instead of 'appendRow', you could use 'addRowButton'. This makes the code more readable and understandable.